<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gideros</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*
QSplitter::handle {
  background: #e8e8e8;
}

QMainWindow::separator {
  background: #e8e8e8;
}
*/</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuRecent_Projects">
     <property name="title">
      <string>Recent Projects</string>
     </property>
     <addaction name="actionProject1"/>
     <addaction name="actionProject2"/>
     <addaction name="actionProject3"/>
     <addaction name="actionProject4"/>
     <addaction name="actionProject5"/>
     <addaction name="actionProject6"/>
     <addaction name="actionProject7"/>
     <addaction name="actionProject8"/>
     <addaction name="actionProject9"/>
     <addaction name="actionProject10"/>
     <addaction name="actionProject11"/>
     <addaction name="actionProject12"/>
     <addaction name="actionProject13"/>
     <addaction name="actionProject14"/>
     <addaction name="actionProject15"/>
    </widget>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="menuRecent_Projects"/>
    <addaction name="actionClose_Project"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_Project"/>
    <addaction name="actionSave_All"/>
    <addaction name="separator"/>
    <addaction name="actionExport_Project"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionReplace"/>
    <addaction name="actionFind_Next"/>
    <addaction name="actionFind_Previous"/>
    <addaction name="separator"/>
    <addaction name="actionFind_in_Files"/>
    <addaction name="separator"/>
    <addaction name="actionGo_To_Line"/>
    <addaction name="actionFold_Unfold_All"/>
    <addaction name="actionFold_Unfold_Top"/>
    <addaction name="actionAuto_indent"/>
   </widget>
   <widget class="QMenu" name="menuCompile">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionCheck_Syntax"/>
    <addaction name="actionCheck_Syntax_All"/>
    <addaction name="actionClear_Output"/>
    <addaction name="actionLive_syntax_checking"/>
   </widget>
   <widget class="QMenu" name="menuPlayer">
    <property name="title">
     <string>&amp;Player</string>
    </property>
    <addaction name="actionStart_Player"/>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="actionStartAll"/>
    <addaction name="separator"/>
    <addaction name="actionPlayer_Settings"/>
    <addaction name="actionLocalhostToggle"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionDeveloper_Center"/>
    <addaction name="actionHelp_Support"/>
    <addaction name="actionDocumentation"/>
    <addaction name="actionAPI_Documentation"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Gideros_Studio"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionUI_Theme"/>
    <addaction name="actionEditor_Theme"/>
    <addaction name="actionUI_and_Editor_Theme"/>
    <addaction name="actionReset_UI_and_Editor_Theme"/>
    <addaction name="separator"/>
    <addaction name="actionFullscreen"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuCompile"/>
   <addaction name="menuPlayer"/>
   <addaction name="menuHelp"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionSave_All"/>
   <addaction name="actionExport_Project"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionToggle_Bookmark"/>
   <addaction name="actionNext_Bookmark"/>
   <addaction name="actionPrevious_Bookmark"/>
   <addaction name="actionClear_Bookmarks"/>
   <addaction name="separator"/>
   <addaction name="actionStart_Player"/>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
   <addaction name="actionStartAll"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose_Project">
   <property name="text">
    <string>Close Project</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionFile_Associations">
   <property name="text">
    <string>File Associations...</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start</string>
   </property>
   <property name="shortcut">
    <string>F5, Ctrl+R</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionStart_Player">
   <property name="text">
    <string>Start Local Player</string>
   </property>
   <property name="toolTip">
    <string>Starts Gideros Player.

When clicked, Gideros Player will start locally and show the 
IP addresses it binds to.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+P</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="text">
    <string>Save All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionToggle_Bookmark">
   <property name="text">
    <string>Toggle Bookmark</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F2</string>
   </property>
  </action>
  <action name="actionNext_Bookmark">
   <property name="text">
    <string>Next Bookmark</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionPrevious_Bookmark">
   <property name="text">
    <string>Previous Bookmark</string>
   </property>
   <property name="shortcut">
    <string>Shift+F2</string>
   </property>
  </action>
  <action name="actionClear_Bookmarks">
   <property name="text">
    <string>Clear Bookmarks</string>
   </property>
  </action>
  <action name="actionCheck_Syntax">
   <property name="text">
    <string>Check Syntax</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+B</string>
   </property>
  </action>
  <action name="actionCheck_Syntax_All">
   <property name="text">
    <string>Check Syntax All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionCancel">
   <property name="text">
    <string>Cancel</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionProject1">
   <property name="text">
    <string>project1</string>
   </property>
  </action>
  <action name="actionProject2">
   <property name="text">
    <string>project2</string>
   </property>
  </action>
  <action name="actionProject3">
   <property name="text">
    <string>project3</string>
   </property>
  </action>
  <action name="actionProject4">
   <property name="text">
    <string>project4</string>
   </property>
  </action>
  <action name="actionProject5">
   <property name="text">
    <string>project5</string>
   </property>
  </action>
  <action name="actionPlayer_Settings">
   <property name="text">
    <string>Player Settings...</string>
   </property>
   <property name="toolTip">
    <string>Opens the Gideros Player IP settings window.

Enter the IP address of the Gideros Player, as read 
on the screen. When Gideros Studio connects to
Gideros Player, Play button is enabled.</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>Replace...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionFind_Next">
   <property name="text">
    <string>Find Next</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionFind_Previous">
   <property name="text">
    <string>Find Previous</string>
   </property>
   <property name="shortcut">
    <string>Shift+F3</string>
   </property>
  </action>
  <action name="actionGo_To_Line">
   <property name="text">
    <string>Go To Line...</string>
   </property>
   <property name="toolTip">
    <string>Go To Line</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionOutput_Panel">
   <property name="text">
    <string>Output Panel</string>
   </property>
  </action>
  <action name="actionLibrary_Manager">
   <property name="text">
    <string>Library Manager</string>
   </property>
  </action>
  <action name="actionExport_Project">
   <property name="text">
    <string>Export Project...</string>
   </property>
   <property name="toolTip">
    <string>Exports current project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionFind_in_Files">
   <property name="text">
    <string>Find in Files...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="actionPreview">
   <property name="text">
    <string>Preview</string>
   </property>
  </action>
  <action name="actionStart_Page">
   <property name="text">
    <string>Start Page</string>
   </property>
  </action>
  <action name="actionAbout_Gideros_Studio">
   <property name="text">
    <string>About Gideros Studio...</string>
   </property>
  </action>
  <action name="actionExport_Pack">
   <property name="text">
    <string>Export Pack...</string>
   </property>
  </action>
  <action name="actionDeveloper_Center">
   <property name="text">
    <string>Developer Center</string>
   </property>
  </action>
  <action name="actionHelp_Support">
   <property name="text">
    <string>Help &amp;&amp; Support</string>
   </property>
  </action>
  <action name="actionAPI_Documentation">
   <property name="text">
    <string>API Documentation</string>
   </property>
  </action>
  <action name="actionProject6">
   <property name="text">
    <string>project6</string>
   </property>
  </action>
  <action name="actionProject7">
   <property name="text">
    <string>project7</string>
   </property>
  </action>
  <action name="actionProject8">
   <property name="text">
    <string>project8</string>
   </property>
  </action>
  <action name="actionProject9">
   <property name="text">
    <string>project9</string>
   </property>
  </action>
  <action name="actionProject10">
   <property name="text">
    <string>project10</string>
   </property>
  </action>
  <action name="actionProject11">
   <property name="text">
    <string>project11</string>
   </property>
  </action>
  <action name="actionProject12">
   <property name="text">
    <string>project12</string>
   </property>
  </action>
  <action name="actionProject13">
   <property name="text">
    <string>project13</string>
   </property>
  </action>
  <action name="actionProject14">
   <property name="text">
    <string>project14</string>
   </property>
  </action>
  <action name="actionProject15">
   <property name="text">
    <string>project15</string>
   </property>
  </action>
  <action name="actionLocalhostToggle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Localhost Toggle</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionClear_Output">
   <property name="text">
    <string>Clear Output</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+X</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionStartAll">
   <property name="text">
    <string>Start All</string>
   </property>
   <property name="toolTip">
    <string>Start to all players</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+P</string>
   </property>
  </action>
  <action name="actionUI_Theme">
   <property name="text">
    <string>Load UI Theme</string>
   </property>
  </action>
  <action name="actionEditor_Theme">
   <property name="text">
    <string>Load Editor Theme</string>
   </property>
  </action>
  <action name="actionUI_and_Editor_Theme">
   <property name="text">
    <string>Load UI and Editor Theme</string>
   </property>
  </action>
  <action name="actionReset_UI_and_Editor_Theme">
   <property name="text">
    <string>Reset UI and Editor Theme</string>
   </property>
  </action>
  <action name="actionFold_Unfold_All">
   <property name="text">
    <string>Fold/Unfold All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+L</string>
   </property>
  </action>
  <action name="actionFold_Unfold_Top">
   <property name="text">
    <string>Fold/Unfold Top</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+T</string>
   </property>
  </action>
  <action name="actionMacro_Support">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Macro Support</string>
   </property>
  </action>
  <action name="actionAuto_indent">
   <property name="text">
    <string>Auto indent</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionLive_syntax_checking">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Live syntax checking</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="text">
    <string>Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
